app/models/grda_warehouse/utility.rb

```diff
@@ -8,7 +8,7 @@
 
 class GrdaWarehouse::Utility
   def self.clear!
-    raise 'Refusing to wipe a production warehouse' if Rails.env.production?
+    raise "Refusing to wipe a #{Rails.env.inspect} warehouse" unless Rails.env.test?
```

Comment:
I can't imagine you'd ever want to run this outside of the test env

Reply:
That's probably safer, thanks.